src/group_theory/group_action/basic.lean

```diff
@@ -192,6 +192,9 @@ local attribute [instance] orbit_rel
 
 variables {α} {β}
 
+lemma orbit_rel_r_apply (x y : β) :
+  @setoid.r _ (orbit_rel α β) x y ↔ x ∈ orbit α y := iff.rfl
```

Comment:
It *might* be more canonical to use `setoid.rel` here, though I'm not sure. `x` and `y` should be implicit either way.

Reply:
yeah, I think that's best. hopefully one day we have a quotient library without 3 variations on everything :b